modules/xmpp/SDP.js

```diff
@@ -366,6 +366,37 @@ SDP.prototype.toJingle = function(elem, thecreator) {
                 });
             }
 
+            const ridLines = SDPUtil.findLines(this.media[i], 'a=rid');
+
+            if (ridLines.length) {
+                // Map a line which looks like "a=rid:2 send" to just
+                // the rid ("2")
```

Comment:
I really like this comment. It makes it clear what the lines directly below are doing and why.